src/Debug.js

```diff
@@ -63,3 +73,61 @@ export function debugNode(node, indentLength = 2) {
 export function debugNodes(nodes) {
   return nodes.map(debugNode).join('\n\n\n');
 }
+
+export function debugInst(inst, indentLength = 2) {
+  if (typeof inst === 'string' || typeof inst === 'number') return escape(inst);
+  if (!inst) return '';
```

Comment:
@ljharb `<div>{Symbol('foo')}</div>` also renderes an empty div. I think this function isn't being used exactly how you imagine it.  Though i'm not sure I event 100% understand it... lol